src/validator/CreditCardValidatorProject.java

```diff
@@ -0,0 +1,60 @@
+package validator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+public class CreditCardValidatorProject {
+	
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
+	private static CreditCardValidator validator = new CreditCardValidator();
+	private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	
+	public static void main(String[] args) {
+        System.out.println("CREDIT CARD VALIDATOR\n");
+        
+        int vendorsLength = vendors.size();
+        int selected = 0;
+        
+        while (true){
+        	printMenu();
+        	try{
+                selected = Integer.parseInt(br.readLine());
+                if(selected < vendorsLength){
+    	        	System.out.println("Type your credit card number:");
+    	        	String cardNumber;
+    				try {
```

Comment:
Try-catch blocks should not be nested. Please try to extract it to the new method. Or maybe it will disappear after change to the rest microservice because BufferedReader will not be used anymore. But it is worth keeping in mind.